eng/tox/tox.ini

```diff
@@ -156,10 +156,10 @@ setenv =
 deps =
   {[base]deps}
   mypy=={[testenv:mypy]mypy_version}
-  types-chardet==4.0.3
-  types-requests==2.27.9
-  types-six==1.16.10
-  types-redis==4.3.21.2
+  types-chardet==5.0.1
```

Comment:
chose versions that said tested on mypy 1.4.1 --- could pin all the most recent versions though -- or not pin versions? @kristapratico

Reply:
I think we want to pin versions to not have surprises... given that next-mypy was running the latest versions, I think let's pin to the latest for all in both mypy and next-mypy